filter/condition/greater.go

```diff
@@ -0,0 +1,21 @@
+package condition
+
+// Greater compares the received value.
+// Returns True if received is greater than threshold.
+type Greater struct {
```

Comment:
Make this private to keep the namespace a bit cleaner than it currently is, and explain the main functionality on `NewGreater`